### ISSUE TITLE
Occupied Location unit spawns and patrols

### ISSUE THREAD

**Issue by DrHardSlump** [nicholasclark-artist/DCG#72]
**Arma 3 Version:** `last` stable
**CBA Version:** `last`
**DCG Version:** `3.1.0.6`

**Server Mods:**
- @CBA_A3
- @dcg
- @RHS
- @VSM
- @ACE
- @Chernarus, Takista, etc...
- custom serverconfig with RHS units

**Description:**
- In our server play more than 10 people simultaneos. I change the enemy count on villages/cities and capital (20/40/80) to compensate the player count in serverconfig.
The problem is all spawn in the same position and stay there. This is too easy and not realistic. Maybe you can change to groups not units and this can patrol the area?

**Steps to reproduce:**
- Change the enemy count on occupy position to max and then attack some village/city/capital

**Where did the issue occur?**
- Dedicated

**Comment by nicholasclark-artist** [comment]
I can't replicate this issue. During my tests, I raised the infantry counts to your numbers and observed the spawns in several locations. The units patrolled in all cases. How often does this issue occur for you? What value did you set for dcg_occupy_locationCount?

Regardless, your capital infantry value seems high. This number doesn't include garrisoned units, static emplacements or vehicles, so in some cases, you're going to see over 100 units per capital. This may negatively affect your server's performance to the extent of causing weird behavior.

**Comment by DrHardSlump** [comment]
class dcg_occupy_locationCount {
	typeName = "SCALAR";
	typeDetail = "";
	value = 1;

Maybe is for the map (Takistan)

**Comment by Radeni** [comment]
@DrHardSlump I noticed it is map dependable because of buildings in some maps. I play DCG on Tanoa and its great,enemy is spread out trough the whole village/city while on Sahrani i had them all in the middle of the city. One thing i did notice that could be changed is the grouping of AI. Usually most of them are in a single group combined with solo AI spawned inside buildings. It would be great if it was changed so that there are at least 2 or 3 main groups patrolling the locations instead of just 1, i noticed this when i was using zeus.
